src/crsAdapter/MerlinAdapter.js

```diff
@@ -206,7 +203,7 @@ class MerlinAdapter {
     getMarkedServiceForServiceType(xmlServices, serviceType) {
         let markedService = void 0;
 
-        xmlServices.some((xmlService) => {
+        xmlServices.forEach((xmlService) => {
             if (xmlService.KindOfService !== CONFIG.crs.serviceTypes[serviceType]) {
                 return;
             }
```

Comment:
What do you think about shorter syntax:
`if (xmlService.KindOfService !== CONFIG.crs.serviceTypes[serviceType]) return`

Reply:
if it is only used for early returns I will try it. but most of the time I do not like it because it is less obvious what happens in case of the condition is true